src/Pacman/index.js

```diff
@@ -73,13 +73,22 @@ export default class Pacman extends Component {
             <Monster key={id} {...props} {...monster} />
         ));
 
+        /**
+         * Render null if user won, otherwise render food, monsters and player
+         */
+        const foodComponent = this.state.ended ?
```

Comment:
This can be wrapped in a React fragment instead of a `div`. Also it doesn't make much sense to call it `foodComponent` as it also includes the monsters.